tests/python_client/testcases/test_query.py

```diff
@@ -654,6 +640,268 @@ def test_query_expr_json_contains_combined_with_normal(self, enable_dynamic_fiel
             res = collection_w.query(expression)[0]
             assert len(res) == limit // 2
 
+    @pytest.mark.tags(CaseLabel.L1)
+    def test_query_expr_all_datatype_json_contains_all(self, enable_dynamic_field):
```

Comment:
Add a case to cover special character.

Reply:
special character has been added in test_query_expr_json_contains_invalid_type.